test/models/issue_test.rb

```diff
@@ -271,6 +271,18 @@ class IssueTest < ActiveSupport::TestCase
     assert issues.all? { |issue| issue.comments.any? { |c| c.text =~ /wat/i } }
   end
 
+  test 'by scheduled_at' do
+    run_to_add_issue = runs :boom_on_atahualpa
+
+    run_to_add_issue.issues << Issue.new(status: 'pending')
+
+    run_to_query = runs :past_ls_on_atahualpa
+
+    issues = Issue.by_scheduled_at("#{(run_to_query.scheduled_at - 1.minutes).strftime('%d/%m/%Y%k:%M')} - #{(run_to_query.scheduled_at + 1.minutes).strftime('%d/%m/%Y%k:%M')}")
```

Comment:
Seba, fijate si se puede poner algo así: 

```suggestion
           from_date = (run_to_query.scheduled_at - 1.minutes).strftime('%d/%m/%Y%k:%M')
282     to_date   = (run_to_query.scheduled_at + 1.minutes).strftime('%d/%m/%Y%k:%M')
283 
284     issues = Issue.by_scheduled_at("#{from_date} - #{to_date}")
```